plaso/output/dynamic.py

```diff
@@ -259,14 +153,14 @@ class DynamicOutputModule(shared_dsv.DSVOutputModule):
       'message', 'parser', 'display_name', 'tag']
 
   def __init__(self, output_mediator):
-    """Initializes an output module object.
+    """Initializes a dynamic delimeter separated values output module object.
```

Comment:
This is a typo, the word is "delimiter" not "delimeter". Please fix the other instances of this as well, it seems this was missed in the past.

Reply:
ack, replace 4 instance across the entire code base